packages/manager/apps/dedicated/client/app/dedicatedCloud/datacenter/backup/delete/delete.module.js

```diff
@@ -0,0 +1,14 @@
+import angular from 'angular';
```

Comment:
import '@ovh-ux/ng-translate-async-loader';
import 'ovh-api-services';
import 'ovh-ui-angular';
import 'angular-translate';
```suggestion
import angular from 'angular';
import '@ovh-ux/ng-translate-async-loader';
import 'ovh-api-services';
import 'ovh-ui-angular';
import 'angular-translate';
```